content/opt/run

```diff
@@ -19,6 +19,10 @@ chown -R rundeck:rundeck /var/rundeck
 chown -R rundeck:adm /var/log/rundeck
 chown -R rundeck:rundeck /var/lib/rundeck
 chown -R rundeck:rundeck /opt/rundeck-defaults
+rm -rf /tmp/rundeck
```

Comment:
Thanks!  Then it will work without having to rm -rf and mkdir /tmp/rundeck as the chmod is done by root unless you're setting --user or --group  

I can do some more investigation if it doesn.  Can you remove those two lines for now as everything else in this PR is ready to go 😃